### PR TITLE
docs(snap): Remove usage instructions from readme

### PR DESCRIPTION
**Pull request by farshidtz**
Usage instructions are being moved to docs; see https://github.com/edgexfoundry/edgex-docs/pull/751.
The snap/README has been updated to include development docs only.

Depends on:
- https://github.com/edgexfoundry/edgex-docs/pull/751

Signed-off-by: Farshid Tavakolizadeh <farshid.tavakolizadeh@canonical.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [x] I have opened a PR for the related docs change (if not, why?)

### CODE REVIEW

File: snap/README.md [hunk 1]
```diff
@@ -1,212 +1,38 @@
...
-and thus this behavior allows the profile `configuration.toml` files in [`$SNAP_DATA`](https://snapcraft.io/docs/environment-variables) to be modified before the service is first started.
-
-This behavior can be overridden by setting the `autostart` configuration setting to `true`. 
-This is useful when configuration and/or device profiles are being provided via configuration or gadget snap content interface.
-
-**Note** - this option is typically set from a gadget snap.
+Please refer to [snapcraft overview](https://snapcraft.io/docs/snapcraft-overview) for more details.
 
+### Obtain a Secret Store token
+The `edgex-secretstore-token` snap slot makes it possible to automatically receive a token from a locally installed platform snap.
```
Comment (MonicaisHer) [inline snap/README.md:24]: How about adding the below link for explaining snap slot?
https://snapcraft.io/docs/interface-management

File: snap/README.md [hunk 2]
```diff
@@ -1,212 +1,38 @@
...
+[![edgex-app-service-configurable](https://snapcraft.io/edgex-app-service-configurable/badge.svg)][edgex-app-service-configurable]
 
-This folder contains snap packaging for the EdgeX Foundry's App Service Configurable application service.
+This directory contains the snap packaging of the EdgeX App Service Configurable application service.
 
-The project maintains a rolling release of the snap on the `edge` channel that is rebuilt and published at least once daily.
+The snap is built automatically and published on the Snap Store as [edgex-app-service-configurable].
 
-The snap currently supports both `amd64` and `arm64` platforms.
+For usage instructions, please refer to [docs].
```
Comment (MonicaisHer) [inline snap/README.md:8]: For usage instructions including installation, configuration, managing Services, debugging, and so on, please refer to docs.

File: snap/README.md (hunk 1 above)
Comment (farshidtz) [reply snap/README.md:24]: These are being added to docs: https://github.com/edgexfoundry/edgex-docs/pull/751
There is a reference to the relevant section few lines below. Once That PR is merged, the hyperlink and anchor will point the user to the right place.

File: snap/README.md (hunk 2 above)
Comment (farshidtz) [reply snap/README.md:8]: These all loosely fall under "usage". Once the reader goes to docs, they will find the details.

**Review by MonicaisHer — APPROVED** [review]

**Pull request merged**